packages/app/features/dashboard/profile/textEllipsis.tsx

```diff
@@ -37,7 +37,7 @@ export function TextEllipsis({ text }: { text: string }) {
       }}
     >
       <Text
-        style={tw.style("text-xs font-bold absolute leading-5")}
+        style={tw.style("text-xs font-bold absolute")}
```

Comment:
why not className?

Reply:
Good catch. I was probably experimenting with `tw.style`  and forgot to return it to className